sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/Sonar.java

```diff
@@ -21,8 +21,15 @@
 
 public class Sonar extends Artifact {
 
+  /**
+   * @deprecated use {@link Release ltaVersion} instead
```

Comment:
```suggestion
   * @deprecated use {@link Sonar#ltaVersion} instead
```

Reply:
We probably want to deprecate the getter and setter too, so it raises an issue when used in the code